docs/source/api.rst

```diff
@@ -42,6 +42,23 @@ Chalice
          app = Chalice(app_name="appname")
          app.debug = True
 
+    .. attribute:: json_encoder
```

Comment:
Could you move this down to the `Response` section of the API docs. This is in the `Chalice` section which will not have a `json_encoder` property.